FreeRTOS/iot-reference-arm-corstone3xx
# Update TF-M to v2.0.0

Description
-----------


The TF-M dependency is upgraded to the recently released version 2.0.0.
See individual commit messages for more info.

Test Steps
-----------

1. Build applications
2. Run applications

Checklist:
----------



Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
